docs/contributing.md

```diff
@@ -23,9 +23,7 @@ Pull requests are the best way to propose changes to the codebase (we use [Githu
 
 ## Running the app locally
 
-```
-yarn start
-```
+```yarn start```
```

Comment:
When I actually pulled this in from the styleguidist aka design-system site, it complained until I got rid of the newlines. Your syntax is valid but whatever plugin they're using must not like it lol